src/components/Countries.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCountries } from '../redux/countries/countriesSlice';
+import Country from './Country';
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  const { allCountries, status } = useSelector((store) => store.countries);
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(getCountries());
+    }
+    window.scrollTo(0, 0);
+  }, [status, dispatch]);
+
+  return (
+    <section className="container-fluid py-5" id="custom-cards">
+      <h2 className="pb-2 border-bottom">Countries</h2>
+      <div className="stats_grid">
+        {
+          allCountries.map((country) => (
+            <Country
+              key={country.countryCode}
+              name={country.name}
+              code={country.countryCode}
+            />
+          ))
+        }
+      </div>
+    </section>
+  );
```

Comment:
- Great job mapping a few countries in your project. 
- I would like to kindly suggest that we stick to the design guidelines on behance as much as possible. On behance, the layout uses grid on mobile screen, with two countries next to each other. But in your project, each country is displayed singly like a stack. Can we fix this? 🌸 

Here is what i can see compared to Figma 👇🏽 
| Your Design  |Behance Design |
| :---:   | :---: | 
| <img width="391" alt="Screenshot 2023-03-17 at 18 22 00" src="https://user-images.githubusercontent.com/99246359/226070676-3f767b53-470c-4e13-a5ac-3bdfc519aa09.png"> |<img width="200" alt="Screenshot 2023-03-17 at 18 42 03" src="https://user-images.githubusercontent.com/99246359/226070724-0716f896-3d97-4676-849c-febc30dc3682.png">  |

Reply:
Hello @Favourezeugwa thank you for the remark, I think the app onrender was not up to date, I just updated it. Please is it possible for you to check and give your comments on that design.

Reply:
Thank you @Favourezeugwa did the necessary corrections and update the code on render too.